### PR TITLE
Update `context.js` and `constants.js`

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

**Review by msporny — APPROVED** [review]

**Review by davidlehn — APPROVED** [review]

File: js/context.js [hunk 1]
```diff
@@ -5,9 +5,68 @@
...
+        id: '@id',
+        type: '@type',
+
+        statusListCredential: {
+          '@id':
+            'https://w3id.org/vc/status-list#statusListCredential',
+          '@type': '@id'
+        },
+        statusListIndex:
+          'https://w3id.org/vc/status-list#statusListCredential'
```
Comment (dlongley) [inline js/context.js]:
```suggestion
          'https://w3id.org/vc/status-list#statusListIndex'
```

**Review by dlongley — CHANGES_REQUESTED** [review]

File: js/context.js (hunk 1 above)
Comment (dlongley) [reply js/context.js]: This appears to be correct over here: https://github.com/w3c-ccg/vc-status-list-2021/blob/main/contexts/v1.jsonld#L66

I recommend doing a diff to make sure human eyes didn't miss anything else.

Comment (JSAssassin) [reply js/context.js]: Thank you for catching that, will do.

Comment (JSAssassin) [reply js/context.js]: Fixed here 2f51823

**Review by dlongley — APPROVED** [review]

**Review by dmitrizagidulin — APPROVED** [review]
:+1: 
Thanks @JSAssassin!

**Comment by davidlehn** [comment]
Re syncing js and json issues: I think the theory of this newer template style is that developers should just edit the js and use the `serialize` script to generate and commit the JSON.  @dmitrizagidulin should confirm.  I assume the generated JSON is supposed to be in git?  The `build` script is going to run `serialize` anyway so I think the main purpose for the JSON in the repo is for local testing?

**Comment by dmitrizagidulin** [comment]
@davidlehn - good question. Yes, the generated JSON is supposed to be committed to Git. The reason for that is -- for many of the contexts this template serves, the w3id URL resolves to that very same generated JSON file, hosted via Github Pages.
Meaning, the main purpose for the JSON is not for local testing, but for serving the context on the web.

**Comment by davidlehn** [comment]
@dmitrizagidulin Ah yes, I forgot that use case.  Would be good to figure out how to write a github action that checks to make sure js and json are in sync.  As was seen here, that can be an issue and is difficult to spot.

**Comment by davidlehn** [comment]
Added tracking issue: https://github.com/digitalbazaar/context-template/issues/6

**Pull request merged**